spec/controllers/users_controller_spec.rb

```diff
@@ -6,6 +6,9 @@
       @user = FactoryGirl.create(:user)
   end
 
+  it { should use_before_filter(:find_user) }
+  it { should_not use_before_filter(:set_user) }
```

Comment:
No, we don't.
It is just a little check if  'use_before_filter' works properly.